components/Entry/index.tsx

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+import Link from 'next/link';
+import Skeleton from 'react-loading-skeleton';
+import styles from './Entry.module.css';
+import { Button, Tag } from '@moai/core';
+import { HiOutlineExternalLink as externalLink } from 'react-icons/hi';
+
+const linkSymbol = <svg width="14px" height="14px" viewBox="0 0 24 24" className="css-1ctnorc"><g id="external_link" className="icon_svg-stroke" stroke="#666" strokeWidth="1.5" fill="none" fillRule="evenodd" strokeLinecap="round" strokeLinejoin="round"><polyline points="17 13.5 17 19.5 5 19.5 5 7.5 11 7.5"></polyline><path d="M14,4.5 L20,4.5 L20,10.5 M20,4.5 L11,13.5"></path></g></svg>;
+
+const getHostName = (url) => {
+  const [host, _] = url.replace(/https?:\/\//, '').split('/');
+  return host;
+}
+
+export const LoadingEntry = () => {
+  return (
+    <div style={{ marginBottom: '32px', padding: '22px', background: '#FFFFFF', borderRadius: 'var(--radius-1)' }}>
+      <h1><Skeleton /></h1>
+      <Skeleton count={5} />
+    </div>
+  )
+}
+export const Entry = ({ doc }) => {
+  return (
+    <div style={{ marginBottom: '32px', padding: '22px', background: '#FFFFFF', borderRadius: 'var(--radius-1)' }}>
```

Comment:
Should we move style to CSS?

Reply:
yeah, thanks for reminding, I will move it to module.css

Reply:
Apparently the right way to do is to use Moai's `Pane` component, but that pane don't support customizing the border and border radus, so I use this instead.

The right way to do in my opinion would be either customize Moai's Pane and use it here, or create a new Pane component and also use it here.